instrument_registry/api/routes/urn.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from fastapi import APIRouter
+from fastapi.responses import JSONResponse
+from instrument_registry.lifespan import cached_data
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+
+# Optional parameter 'version' is included, but not used. In a new ticket
+# versioning of instruments should be handled.
+@router.get("/")
+async def get_instrument(urn: str, version: str = "latest") -> JSONResponse:
```

Comment:
it would be AMAZING if you could make the URN type a literal type so all options become available in the openapi specs :D 

something like this:
urn: Literal[tuple(cacheddata[somefield])]

Reply:
It would indeed be AMAZING :D The thing is that the URN's are strings that are determined during runtime, and we don't see how we can use this in the Literal type hint. Do you have any suggestions?

Reply:
i would assume the example above would work. you can also have a look at this:
https://docs.pydantic.dev/latest/concepts/models/#dynamic-model-creation

Reply:
i had a quick look and could not get it to work.

best i had was this:
```
def validate_urn(urn: str) -> str:
    if urn not in MYLIST:
        raise HTTPException(status_code=400, detail=f"Invalid urn: {urn}")
    return urn

@router.get("/test")
async def get_urns(urn: str = Depends(validate_urn)) -> JSONResponse:
    return JSONResponse(content=urn)
    ```